models.py

```diff
@@ -0,0 +1,350 @@
+        db_cursor.execute(new_ride)
+        db_connection.commit()
+
+    @classmethod
+    def create_ride(cls, ride, driver_id, departuretime, numberplate, maximum, status="pending"):
+        """Creates a new ride"""
+
+        cls(ride, driver_id, departuretime, numberplate, maximum, status)
+        return make_response(jsonify({"message" : "ride has been successfully created"}), 201)
+
+    @staticmethod
+    def update_ride(ride_id, ride, driver_id, departuretime, numberplate,
+                    maximum):
+        """Updates ride information"""
+        db_connection = conn()
+        db_cursor = db_connection.cursor()
+        db_cursor.execute("SELECT * FROM rides")
+        rides = db_cursor.fetchall()
+        for ride in rides:
+            if ride[0] == ride_id:
+                db_cursor.execute("UPDATE rides SET ride=%s, driver_id=%s, departuretime=%s, numberplate=%s, maximum=%s WHERE ride_id=%s",
+                                  (ride, driver_id, departuretime, numberplate, maximum, ride_id))
+                db_connection.commit()
+                db_connection.close()
+
+                return make_response(jsonify({"message" : "ride has been successfully updated"}), 200)
+        return make_response(jsonify({"message" : "ride does not exist"}), 404)
+
+    @staticmethod
+    def start_ride(ride_id, driver_id):
+        """starts a ride"""
+        db_connection = conn()
+        db_cursor = db_connection.cursor()
+        db_cursor.execute("SELECT * FROM rides WHERE ride_id=%s", (ride_id,))
+        ride = db_cursor.fetchall()
+        if ride != []:
+            ride = ride[0]
+            if int(ride[2]) == driver_id:
+                db_cursor.execute("UPDATE rides SET status=%s WHERE ride_id=%s", ("given", ride_id))
+                db_connection.commit()
+                db_connection.close()
+
+                return {"message" : "ride has started"}
+
+            return {"message" : "The ride you want to start is not your ride."}
+
+        return {"message" : "ride does not exist"}
+
+    @staticmethod
+    def delete_ride(ride_id):
+        """Deletes a ride"""
+        db_connection = conn()
+        db_cursor = db_connection.cursor()
+        db_cursor.execute("SELECT * FROM rides")
+        rides = db_cursor.fetchall()
+
+        for ride in rides:
+            if ride[0] == ride_id:
+                db_cursor.execute("DELETE FROM rides WHERE ride_id=%s", (ride_id,))
+                db_connection.commit()
+                db_connection.close()
+
+                return make_response(jsonify({"message" : "ride has been successfully deleted"}), 200)
+        return make_response(jsonify({"message" : "user does not exists"}), 404)
+
+    @staticmethod
+    def get_ride(ride_id):
+        """Gets a particular ride"""
+        db_connection = conn()
+        db_cursor = db_connection.cursor()
+        db_cursor.execute("SELECT * FROM rides WHERE ride_id=%s", (ride_id,))
+        ride = db_cursor.fetchall()
```

Comment:
Why would you do a `fetchall` here given that you only need on value? `fetchall` is an expensive operation.